src/backend/views/components/sidebar.twig

```diff
@@ -0,0 +1,54 @@
+
+{% set mainClass = 'docs-sidebar' %}
+
+<div data-module="sidebar" class="{{mainClass}}__wrapper">
```

Comment:
Wrong usage of BEM: Element should not include its Block

```
<docs-sidebar__wrapper>
  <docs-sidebar/>
</docs-sidebar>
```

The Block should include Elements instead
```
<docs-sidebar>
  <docs-sidebar__toggler/>
  <docs-sidebar__body/>
</docs-sidebar>
```